src/convert/streams.ts

```diff
@@ -223,6 +223,10 @@ export class ZipWriter extends ComponentWriter {
     void pipeline(this.zip, this.getOutputStream());
   }
 
```

Comment:
Is the intent here to always have getter/setter right after the constructor?

Reply:
the default rules are here: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/member-ordering.md